[Bug]: test_base_url_valid_hosted_with_api_key fails for NVIDIA LLMs

### Bug Description

When running the tests under ```llama-index-integrations/llms/llama-index-llms-nvidia```, I got

```
FAILED tests/test_base_url.py::test_base_url_valid_hosted_with_api_key[https://integrate.api.nvidia.com/v1/] - AttributeError: 'NVIDIA' object has no attribute 'base_url'
```

### Version

I'm developing and contributing to LlamaIndex from source.

### Steps to Reproduce

I used the commands from the [quick start guide](https://github.com/run-llama/llama_index/blob/main/CONTRIBUTING.md#-quick-start-guide) to run the tests under ```llama-index-integrations/llms/llama-index-llms-nvidia```.

After cloning the repository and installing ```uv```:
```
uv sync
cd llama-index-integrations/llms/llama-index-llms-nvidia
uv run -- pytest
```

### Relevant Logs/Tracbacks

```shell

```

--- a/llama-index-integrations/llms/llama-index-llms-nvidia/tests/test_base_url.py
+++ b/llama-index-integrations/llms/llama-index-llms-nvidia/tests/test_base_url.py
@@ -45,15 +45,15 @@ def test_base_url_valid_not_hosted(base_url: str, mock_local_models: None) -> No
 
 
 @pytest.mark.parametrize("base_url", ["https://integrate.api.nvidia.com/v1/"])
-def test_base_url_valid_hosted_without_api_key(base_url: str) -> None:
+def test_base_url_valid_hosted_with_api_key(base_url: str) -> None:
     Interface(base_url=base_url, api_key="BOGUS")
 
 
 @pytest.mark.integration
 @pytest.mark.parametrize("base_url", ["https://integrate.api.nvidia.com/v1/"])
-def test_base_url_valid_hosted_with_api_key(base_url: str) -> None:
+def test_base_url_valid_hosted_without_api_key(base_url: str) -> None:
     llm = Interface()
-    assert llm.base_url == base_url
+    assert llm.api_base == base_url
 
     llm = Interface(base_url=base_url)
-    assert llm.base_url == base_url
+    assert llm.api_base == base_url

